Doc/library/os.rst

```diff
@@ -2525,6 +2525,10 @@ features:
    .. versionadded:: 3.5
       Added the :attr:`st_file_attributes` member on Windows.
 
+   .. versionchanged:: 3.5
+      Windows now returns the `file index
+      <https://msdn.microsoft.com/en-us/library/aa363788>`_ as :attr:`st_ino`.
```

Comment:
Not that this is your fault, but...

>The identifier (low and high parts) and the volume serial number uniquely identify a file on a single computer. To determine whether two open handles represent the same file, combine the identifier and the volume serial number for each file and compare them.

That is a naive statement, written without caution and with an unwarranted air of authority. The VSN is not necessarily unique and not necessarily non-zero. The file index may also be zero if it's not supported. Who writes this stuff -- interns? You get a much clearer picture as you go deeper into the system calls and protocol specs, which are documents that are more obviously written by people who understand what they're talking about.

Reply:
Feel free to post that feedback on the page (click "No" at the bottom for whether the page is helpful).

Most of the docs for APIs that have been around since XP (and typically earlier, but XP is the minimum listed in the docs) have assumptions that were almost-always true at the time but are no longer (single drive, single OS, narrow selection of file systems, etc.).